source/openapi-admin-v3.yaml

```diff
@@ -7298,9 +7528,12 @@ components:
 
             - the relationship may point to many foreign documents.
             - the local field must be defined as an array in the collection schema.
+    SchemaDefinition:
+      type: object
+      description: A valid [schema](https://www.mongodb.com/docs/realm/schemas) for the collection.
```

Comment:
Is there a reason we don't document the schema object here? The page we link out to has a "Define a Schema" section with an example object: https://www.mongodb.com/docs/atlas/app-services/schemas/#define-a-schema

It struck me as weird that I can view all the other object details here in the API spec, but not schema.

Reply:
Historically because it's highly dependent on the user's definition. That said there are some invariants that we can include here and use them in a recursive definition - good catch!